archive/archive_asset_lookup/src/endpoint.py

```diff
@@ -0,0 +1,103 @@
+import os
+import tempfile
+import logging
```

Comment:
We tend to just print ... stdout gets logged in lambdas. Worth asking @alexwlchan if there is another reason.

Reply:
mostly there for isomorphism really, I think it'll pick up whatever ambient settings for the logging have been set up (if this were called as a module)

Reply:
Turned out there were various hoops to jump through to get standard logging working in Lambda which are now in the `core-archive-function` branch.